src/cli.lisp

```diff
@@ -44,8 +49,18 @@
        ,@body)))
 
 
-;;; Qi Install ($ qi --install [package] / $ qi -i [package]) internals
+;;; Qi install-deps ($ qi --install-deps project.asd)
+(defun opt-install-deps (input)
+  "Install the dependencies locally for the system definition file provided as INPUT."
+  (let* ((pwd (uiop:getenv "PWD"))
```

Comment:
Is it safe to use `$PWD` like this? Is there a better way to get the directory from which `bin/qi` is run?